### ISSUE TITLE
Jess comment on Paper: should simulate from super population

### ISSUE THREAD

**Issue by mwklose** [chasedlatour/Multiple_outcomes_pregnancy#2]
https://github.com/chasedlatour/Multiple_outcomes_pregnancy/blob/3a484dfac4cd0e135ac61bfc3da5d86869716401/00_data%20generation%20functions.R#L19

Hello Chase! I am getting a chance to look at the draft, and towards Jess's point on being more specific what the ground truth is, I think that all be need to do is run the chunk of code above, with 1 simulation, n=10,000,000. Then, we find the ACE, which is our "ground truth" under that specific DGP. 

I think we need to redo the 10,000,000 for each scenario, so we end up getting 12 "ground truths" to compare each simulation to, then find bias for that. 

Other comments from when I brought this up in Steve's lab meeting: 
- apparently, it is much easier to generate all the potential outcomes, and then assign based on treatment received and whichever occurs first. So, for our example, we would generate time to live birth, time to fetal death, time to preeclampsia, time to censoring, then choose the minimum. I think this makes it more difficult to make specific parameter choices like we've done, but there should be ways around it.

**Comment by chasedlatour** [comment]
Hi Mark!

I saw your other email and haven't had a chance to respond - sorry! Will
get to that later.

So, we ARE currently generating the potential outcomes for the 12 target
populations, but we are doing it differently than you suggested.
Specifically, we're using the potential outcomes from each 5,000 person
cohort, and the ground truth is then the average of each
parameter estimate across the 2,000 ohorts. From my experience with
simulation studies, that's my preferred approach over one 10,000,000 person
sample, though I honestly think they'll give VERY similar answers. I
read Jess's comment as us needing to provide that level of detail. Feel
free to disagree, and we can talk further. I'm also planning to join her
office hours tomorrow and can confirm this with her.

For the suggestion from Steve's group meeting: Yes, I agree that that is an
easier way to approach this, but I don't think it invalidates anything
we've done. I originally had done it that way, actually, but shifted it
after running into some complications (which I honestly don't remember
anymore 🫠). If I had to re-do this, I might do it that way, but there's
honestly no world in which I want to re-write and re-run all of this when I
think we'll get the same thing. Importantly, we still choose the first
outcome from each approach, so we still maintain the causal integrity. A
good note I think to keep in mind for the future!

Chase
